merlin-framework-processor/src/main/java/gov/nasa/jpl/aerie/merlin/processor/metamodel/MissionModelRecord.java

```diff
@@ -13,20 +14,23 @@ public final class MissionModelRecord {
   public final TypeElement topLevelModel;
   public final List<TypeRule> typeRules;
   public final List<ActivityTypeRecord> activityTypes;
+  public final Collection<TypeElement> autoValueMapperRequests;
   public final Optional<ConfigurationTypeRecord> modelConfigurationType;
 
   public MissionModelRecord(
       final PackageElement $package,
       final TypeElement topLevelModel,
       final Optional<ConfigurationTypeRecord> modelConfigurationType,
       final List<TypeRule> typeRules,
-      final List<ActivityTypeRecord> activityTypes)
+      final List<ActivityTypeRecord> activityTypes,
+      final Collection<TypeElement> autoValueMapperRequests)
   {
     this.$package = Objects.requireNonNull($package);
     this.topLevelModel = Objects.requireNonNull(topLevelModel);
     this.modelConfigurationType = Objects.requireNonNull(modelConfigurationType);
     this.typeRules = Objects.requireNonNull(typeRules);
     this.activityTypes = Objects.requireNonNull(activityTypes);
+    this.autoValueMapperRequests = autoValueMapperRequests;
```

Comment:
For consistency I'd either add `Objects.requireNonNull(...)` here or refactor this class to be a `record` (without any `Objects.requireNonNull(...)` checks if you like).

Reply:
Done :+1: